torchvision/prototype/datasets/_builtin/eurosat.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict, List, Tuple
+
+from torchdata.datapipes.iter import IterDataPipe, Mapper
+from torchvision.prototype.datasets.utils import Dataset, DatasetConfig, DatasetInfo, HttpResource, OnlineResource
+from torchvision.prototype.datasets.utils._internal import hint_sharding, hint_shuffling
+from torchvision.prototype.features import EncodedImage, Label
+
+
+class EuroSAT(Dataset):
+    def _make_info(self) -> DatasetInfo:
+        return DatasetInfo(
+            "EuroSAT",
```

Comment:
For now, the canonical dataset names are all lower case

```suggestion
            "eurosat",
```